paper/paper.md

```diff
@@ -75,6 +75,27 @@ bibliography: paper.bib
 # aas-journal: Astrophysical Journal <- The name of the AAS journal.
 ---
 
+# Summary
+
+Connectome Mapper 3 (CMP3) is an open-source processing pipeline software, written in Python 3, for multi-scale multi-modal connectome mapping
```

Comment:
Connectome Mapper 3 (CMP3) is an open-source processing pipeline software (add:) *for neuroimaging data* (...),

Reply:
Maybe add this or something similar: The connectome refers to a representation of the brain as a graph derived from both structural or functional connectivity between a pair of brain regions.